charms/jupyter-controller/reactive/jupyter_controller.py

```diff
@@ -63,7 +78,8 @@ def start_charm():
                     'name': 'jupyter-controller',
                     'command': ['/manager'],
                     'config': {
-                        'USE_ISTIO': 'false',
+                        'USE_ISTIO': hookenv.is_relation_made('service-mesh'),
```

Comment:
The type here is changing from a string to a bool.  I know this is going to be YAML-ified and then probably cast to a string anyway, but just wanted to double-check that there won't be a capitalization issue introduced here?

Reply:
Good catch, that might cause issues:

https://github.com/kubeflow/kubeflow/blob/19da809/components/notebook-controller/controllers/notebook_controller.go#L180

Forcing it to be a lower-case string just to be sure